extensions/mssql/src/ui/dialogBase.ts

```diff
@@ -239,7 +239,8 @@ export abstract class DialogBase<DialogResult> {
 				data: data,
 				columns: columns,
 				width: DefaultTableWidth,
-				height: getTableHeight(data.length, DefaultMinTableRowCount, maxRowCount)
+				height: getTableHeight(data.length, DefaultMinTableRowCount, maxRowCount),
+				forceFitColumns: azdata.ColumnSizingMode.ForceFit
```

Comment:
I am not sure whether this should be the default option, you probably should pass it in as an option.

Reply:
Reverted this change